custom/icds_reports/tasks.py

```diff
@@ -664,6 +670,12 @@ def _agg_ls_vhnd_form(state_id, day):
         AggregateLsVhndForm.aggregate(state_id, force_to_date(day))
 
 
+@track_time
+def _agg_gov_vhnd_form(state_id, day):
```

Comment:
After shifting the aggregation to gov dashboard table. you wont need this function.

Reply:
removed it